src/components/VueCommand.vue

```diff
@@ -9,9 +9,16 @@
       <div
         v-if="!hideBar"
         class="term-bar">
-        <span class="term-title">
-          {{ title }}
-        </span>
+        <template v-if="customBar">
```

Comment:
Instead of this, you could drop the prop `customBar` entirely and do:

```suggestion
        <slot name="customBar">
          <span class="term-title">
            {{ title }}
          </span>
        </slot>
```

This way, the default bar will be used whenever no custom one is provided.

Reply:
Oh yeah, fallback content ! Right on point, thank you !